.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-commonjs */
 module.exports = {
-    extends: ["@khanacademy"],
+    extends: ["./packages/eslint-config-khan/index.js"],
```

Comment:
The benefit of using this here is that running `yarn lint` will run any changes to the config against the code in this repo which gives us some information about how the config is performing before we actually publish changes to it.